toolkit/offchain/src/scripts_data.rs

```diff
@@ -27,19 +26,9 @@ pub struct Addresses {
 	pub version_oracle_validator: String,
 }
 
-/// Hashes of applied validators in partner-chains smart contracts.
-#[derive(Clone, Debug, Default, Serialize, PartialEq, Eq)]
-pub struct ValidatorHashes {
-	pub committee_candidate_validator: MainchainAddressHash,
-	pub d_parameter_validator: MainchainAddressHash,
-	pub illiquid_circulation_supply_validator: MainchainAddressHash,
-	pub permissioned_candidates_validator: MainchainAddressHash,
-	pub reserve_validator: MainchainAddressHash,
-	pub version_oracle_validator: MainchainAddressHash,
-}
-
 /// Policy IDs of applied scripts in partner-chains smart contracts.
 #[derive(Clone, Debug, Default, Serialize, PartialEq, Eq)]
+#[serde(rename_all = "PascalCase")]
```

Comment:
We output snake_case JSON at the moment, but there is no user for the command yet.
I've found his when working on integrating our new commands in local-environment setup